supervision/detection/tools/inference_slicer.py

```diff
@@ -36,6 +36,12 @@ class InferenceSlicer:
             slices in the format `(width_ratio, height_ratio)`.
         iou_threshold (Optional[float]): Intersection over Union (IoU) threshold
             used for non-max suppression.
+        merge_detections (Optional[bool]): Whether to merge the detection from all
+            slices or simply concatenate them. If `True`, Non-Maximum Merging (NMM),
+            otherwise Non-Maximum Suppression (NMS), is applied to the detections.
+        perform_standard_pred (Optional[bool]): Whether to perform inference on the
```

Comment:
Let's not touch `InferenceSlicer` for now. We can add this on step 3 - after we have NMM and weighted boxes methods.

Alternatively, if you make a separate PR, it'd be easier to test in isolation - likely easier to merge too :)

Reply:
For us, continuing with this PR would be best, even if it means removing some code.

At the same time, if you find it easier to make a new one or have started work already - that's fine too :)